framework/application/CMakeLists.txt

```
@@ -78,8 +78,18 @@ common_build_directives(gfxrecon_application)
 if (${RUN_TESTS})
     add_executable(gfxrecon_application_test "")
     target_sources(gfxrecon_application_test PRIVATE
-            ${CMAKE_CURRENT_LIST_DIR}/test/main.cpp)
+            ${CMAKE_CURRENT_LIST_DIR}/test/main.cpp
+            ${CMAKE_CURRENT_LIST_DIR}/../../tools/platform_debug_helper.cpp)
     target_link_libraries(gfxrecon_application_test PRIVATE gfxrecon_application)
+    if (MSVC)
+        # Force inclusion of "gfxrecon_disable_popup_result" variable in linking.
+        # On 32-bit windows, MSVC prefixes symbols with "_" but on 64-bit windows it doesn't.
+        if(NOT CMAKE_CL_64)
```

Use [CMAKE_SIZEOF_VOID_P](https://cmake.org/cmake/help/latest/variable/CMAKE_SIZEOF_VOID_P.html) to distinguish between 32/64.

`CMAKE_CL_64` is discouraged for new projects.

```suggestion
        if(CMAKE_SIZEOF_VOID_P EQUAL 4)
```

---

Nope, `catch2`, and we've been discussing today that popup windows are not blocked.

---

But given switching to ctest would be a big change I'd say this is fine.

---

I'll make the `CMAKE_SIZEOF_VOID_P` change.

---

I suggest leaving that for another PR.

---

Oh, I just did it.
Should I back it out?